### PR TITLE
fix: updates for dead sessions

### PR DESCRIPTION
**Pull request by lewxdev**

### CODE REVIEW

File: app/utils/redis.ts [hunk 1]
```diff
@@ -24,10 +25,15 @@ export async function encodeData(value: number[]) {
...
-export async function getSession(id: string) {
+export async function getSession(sessionId: string): Promise<SocketData> {
   const redis = getClient();
-  const value = await redis.hget(sessionKey, id);
-  return value === "alive" || value === "dead" ? value : null;
+  const sessionState = await redis.hget(sessionKey, sessionId);
+  if (sessionState === "alive" || sessionState === "dead") {
+    return { sessionId, sessionState };
+  }
+  const newSessionId = (sessionId = crypto.randomBytes(8).toString("hex"));
```
Comment (drewradcliff) [inline app/utils/redis.ts]: is it necessary to reassign `sessionId` here? 
```suggestion
  const newSessionId = crypto.randomBytes(8).toString("hex");
```

**Review by drewradcliff — CHANGES_REQUESTED** [review]
nice work bro

File: app/utils/redis.ts (hunk 1 above)
Comment (lewxdev) [reply app/utils/redis.ts]: not at all, just a left over bit of code from moving things around

**Review by drewradcliff — APPROVED** [review]
👍

**Pull request merged**